app/views/users/show.html.erb

```diff
@@ -0,0 +1,135 @@
+<div class="container profile-page">
+  <div class="profile-images">
+    <div class="profile-banner">
+      <div class="profile-info d-flex align-items-center">
+        <div class="profile-pic">
+          <%= gravatar_for(@user, size: 140) %>
+        </div>
+        <%= content_tag :div, @user.name, class: 'profile-name' %>
+        <div class="friend-button">
+          <% if @user != current_user %>
+            <% if current_user.pending_requests_to_users.include?(@user) %>
```

Comment:
**[REQUIRED]**

Let's separate the code logic from the views([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright).